demisto_sdk/commands/create_artifacts/tests/content_creator_test.py

```diff
@@ -185,3 +187,28 @@ def test_copy_file_to_artifacts(self):
         file_path = os.path.join(self._files_to_artifacts_dir, filename)
         content_creator.copy_file_to_artifacts(file_path)
         assert filecmp.cmp(file_path, os.path.join(self.content_repo, filename))
+
+    def test_content_legacy_with_no_md(self):
+        """
+        Given
+        - valid content dir
+        When
+        - copying the content folder to a content bundle(flatten files)
+        Then
+        Ensure no md files were copied to content bundle
+        Ensure md files were copied to packs bundle.
```

Comment:
```suggestion
```

Reply:
Why remove this?

Reply:
sorry, meant to add it as bullet: `- Ensure md files were copied to packs bundle.`